src/pages/NewPasswordPage.tsx

```diff
@@ -0,0 +1,53 @@
+import './NewPasswordPage.css'
+import { useState } from "react";
+import { useFetch } from "../hooks/useFetch";
+import Button from "../components/Button/Button";
+import ButtonBackIcon from "../components/ButtonBackIcon/ButtonBackIcon";
+import PasswordInputWithRequirements from "../components/PasswordInputWithRequirements/PasswordInputWithRequirements";
+import HeroLogo from "../components/HeroLogo/HeroLogo";
+
+const NewPasswordPage = () => {
+  const [password, setPassword] = useState("");
+
+  const url = "http://localhost:8007/user/rpassword/reset";
```

Comment:
this endpoint is not yet operational

Reply:
Yes, this endpoint not appear in the swagger i take from a trello card.